### PR TITLE
Add webcam support and example

### PR DESCRIPTION
**Pull request by sinbycos**
I have provided webcam support with OpenCV Python and display the dominant emotion detection on GUI. I have also provided an example to how to use fermodel using webcam.

### CODE REVIEW

File: EmoPy/src/fermodel.py [hunk 1]
```diff
@@ -91,10 +92,11 @@ def _choose_model_from_target_emotions(self):
         model_indices = [self.emotion_index_map[emotion] for emotion in self.target_emotions]
         sorted_indices = [str(idx) for idx in sorted(model_indices)]
         model_suffix = ''.join(sorted_indices)
-        model_file = 'models/conv_model_%s.hdf5' % model_suffix
-        emotion_map_file = 'models/conv_emotion_map_%s.json' % model_suffix
-        emotion_map = json.loads(open(resource_filename('EmoPy',emotion_map_file)).read())
```
Comment (angelicaperez37) [inline EmoPy/src/fermodel.py]: I advise against removing the use of the resource_filename function in this context. This allows us to access files in a Python package easily without hardcoding entire paths. If you're having trouble with this line, you may have missed the installation step for the EmoPy package described in the README

File: EmoPy/examples/fermodel_example_webcam.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+cascPath = "~/EmoPy/venv/lib/python3.5/site-packages/cv2/data/haarcascade_frontalface_default.xml"
+faceCascade = cv2.CascadeClassifier(cascPath)
+#Specify the camera which you want to use. The default argument is '0'
+video_capture = cv2.VideoCapture(0)
+
+while True:
+	#Capture frame-by-frame
+	ret, frame = video_capture.read()
+	#Save the captured frame on disk
+	file = '~/EmoPy/models/examples/image_data/image.jpg'
```
Comment (angelicaperez37) [inline EmoPy/examples/fermodel_example_webcam.py]: My system wasn't able to access this file when run locally. Similar to my other comment, using relative file paths such that any machine can find them will work best 👍

**Comment by microcosm** [comment]
As you can see in [PR 16](https://github.com/thoughtworksarts/EmoPy/pull/16), @xuv has updated this PR with @angelicaperez37's change requests. We will now accept this PR and then @xuv will apply the fixes on top of that.

Thanks @sinbycos for starting this! And thanks @xuv for adding to it!

**Comment by xuv** [comment]
@all-contributors please add @sinbycos for code and examples

**Pull request merged**